File: run-seed.js
const fs = require('fs');
const { Client } = require('pg');

const seedFilePath = '/Users/abraham/Downloads/seed_hotel_tulum_v3.sql';
const sql = fs.readFileSync(seedFilePath, 'utf-8');

const client = new Client({
  connectionString: 'postgresql://housekeeping:devpassword@localhost:5433/housekeeping',
});

(async () => {
  try {
    await client.connect();
    console.log('✓ Conectado a la BD');

    await client.query(sql);
    console.log('✓ Script ejecutado sin errores');
  } catch (err) {
    console.error('✗ Error al ejecutar script:', err.message);
    process.exit(1);
  } finally {
    await client.end();
  }
})();

File: run-seed.py
#!/usr/bin/env python3
import psycopg2
import sys

# Database connection parameters
db_params = {
    'host': 'localhost',
    'port': 5433,
    'database': 'housekeeping',
    'user': 'housekeeping',
    'password': 'devpassword'
}

sql_file = '/Users/abraham/Downloads/seed_hotel_tulum_v3.sql'

try:
    # Read SQL file
    with open(sql_file, 'r') as f:
        sql_content = f.read()

    print('✓ SQL file read successfully')

    # Connect to database
    conn = psycopg2.connect(**db_params)
    cursor = conn.cursor()
    print('✓ Connected to database')

    # Execute SQL
    cursor.execute(sql_content)
    conn.commit()
    print('✓ Script executed successfully without errors')

    cursor.close()
    conn.close()

except psycopg2.Error as e:
    print(f'✗ Database error: {e}', file=sys.stderr)
    sys.exit(1)
except Exception as e:
    print(f'✗ Error: {e}', file=sys.stderr)
    sys.exit(1)
